.github/workflows/container-structure-test.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       - name: Install Google Container Structure Test Framework
         run: curl -LO https://storage.googleapis.com/container-structure-test/latest/container-structure-test-linux-amd64 && chmod +x container-structure-test-linux-amd64 && sudo mv container-structure-test-linux-amd64 /usr/local/bin/container-structure-test
       - name: Build docker images PHP Only
-        run: docker build -t antistatique/php-dev:${{ matrix.php_version }} -f ./php/${{ matrix.php_version }}/Dockerfile .
+        run: docker build --pull -t antistatique/php-dev:${{ matrix.php_version }} -f ./php/${{ matrix.php_version }}/Dockerfile .
```

Comment:
This is to be sure we use the last PHP docker image and do not have a surprise with their update.

Reply:
seems a good idea yeah !!